monticello/style.css

```diff
@@ -81,4 +81,19 @@ represented in .json
 	opacity: 1;
 }
 
-/* END BUTTON STYLES */
\ No newline at end of file
+/* END BUTTON STYLES */
+/*
```

Comment:
Can we remove these commented styles?

Reply:
I think (based on the conversation above) I'm about to UNcomment them and pull out the block styles.